indexer/road_shields_parser.cpp

```diff
@@ -126,7 +136,17 @@ class RoadShieldParser
       else
       {
         shield = ParseRoadShield(rawText.substr(slashPos + 1));
-        shield.m_type = FindNetworkShield(std::string(rawText.substr(0, slashPos)));
+        // TODO: use a network-based shield type override only if a parser couldn't make it
+        // more specific than country's default shield type.
+        // E.g. "94" is set to Generic_Orange by Estonia parser, but then
+        // is overriden by "ee:national" => Generic_Red.
+        // (can't override just RoadShieldType::Default, as e.g. Russia parser uses Generic_Blue as country default).
+        if (shield.m_type != RoadShieldType::Hidden)
```

Comment:
Please, add/update unit tests for this Estonian logic.

Reply:
done